pkg/mgrconfig/load.go

```diff
@@ -234,9 +237,19 @@ func (cfg *Config) completeServices() error {
 	return nil
 }
 
-func (cfg *Config) initTimeouts() {
+func (cfg *Config) initTimeouts() error {
 	slowdown := 1
 	switch {
+	case cfg.Type == "qemu":
+		// If TCG is enabled for QEMU, increase the slowdown.
+		vmConfig := make(map[string]interface{})
+		if err := json.Unmarshal(cfg.VM, &vmConfig); err != nil {
+			return fmt.Errorf("failed to parse VM config: %w", err)
+		}
+		args, ok := vmConfig["qemu_args"].(string)
```

Comment:
`json.RawMessage` is just `[]byte`:
https://pkg.go.dev/encoding/json#RawMessage
It should be possible to call strings/bytes.Contains on it directly (with a case).
Then also don't need error handling.